Proteomics/workflows/ntails/README.md

```diff
@@ -0,0 +1,43 @@
+# GalaxyP Workflow: N-Tails Data Analysis
```

Comment:
Is this document just a description of the workflow? Or do you want to use it as tutorial? In the latter case, it would be better to put it in the tutorials directory, and to follow the requirements for a tutorial. In the first case, can you remove the boxes, please? They are just used for tutorials not for README